### ISSUE TITLE
Channels (one-to-many talk conversations)

### ISSUE THREAD

**Issue by marcoambrosini** [nextcloud/spreed#13508]
Add ability for any talk user to create channels. In this channel, only the owner (channel creator) can post updates, while others can “join” the channel to **view those posts and react using emojis**. This feature would be useful for both **personal updates** (e.g., sharing life moments, status changes) and **professional communication**, such as posting regular work reports or progress updates on projects.

This would allow for structured communication where channel owners can update their followers without the thread being interrupted by unrelated messages. Participants can stay engaged through reactions, maintaining a sense of connection without interrupting the flow of posts. 

Personal channels could also appear in the profile page. Either as a link to a conversation or as a window into it.

This fits would all fit very nicely with the theme for the next release https://github.com/nextcloud/server/issues/43763

Here's Pavel Durov's public channel on Telegram as an example:

 ![Image](https://github.com/user-attachments/assets/06d96c66-2ada-4ee2-aeb6-320a3e28358a)

**Comment by nickvergessen** [comment]
TLDR: split permissions for chat and reactions and then only give reaction permission

**Comment by SystemKeeper** [comment]
* Ref https://github.com/nextcloud/spreed/issues/11329
